packages/app/src/app/overmind/namespaces/editor/actions.ts

```diff
@@ -100,7 +100,11 @@ export const sandboxChanged: AsyncAction<{ id: string }> = withLoadApp<{
     actions.workspace.openDefaultItem();
   } catch (error) {
     state.editor.notFound = true;
-    state.editor.error = error.message;
+    let detail = error.response?.data?.errors?.detail;
```

Comment:
Again, while it works, I'm not sure it's the best approach to handle this here, but it was the easiest to get the job done. 🙂

Reply:
That's interesting, I was thinking that our error handling logic should solve this and put the error already on `error.message`. Didn't we change something here @christianalfoni?

Reply:
As far as I remember, we always had that 422 error message.

Reply:
Hmyes, but there was a time that we actually showed the error. There has been a regression long ago that made it look like this cryptic error.